crates/solvers/src/infra/config/baseline/file.rs

```diff
@@ -45,7 +45,7 @@ pub async fn load(path: &Path) -> super::Config {
         .await
         .unwrap_or_else(|e| panic!("I/O error while reading {path:?}: {e:?}"));
     let config = toml::de::from_str::<Config>(&data)
-        .unwrap_or_else(|_| panic!("TOML syntax error while reading {path:?}"));
+        .unwrap_or_else(|err| panic!("TOML syntax error while reading {path:?}: {err:?}"));
```

Comment:
This was intentional to not leak secrets in logs.
```suggestion
        .unwrap_or_else(|_| panic!("TOML syntax error while reading {path:?}"));
```

Reply:
So I checked that the only file that contains secrets is the top level driver config (for which I had already reverted the change). I assume your concern is that in the future we may add secrets to the solver specific configs and then forget to change that line?

Reply:
Yes, this is my concern.